src/main/kotlin/com/zepben/evolve/cim/iec61968/common/Location.kt

```diff
@@ -24,18 +24,18 @@ class Location @JvmOverloads constructor(mRID: String = "") : IdentifiedObject(m
     private var _positionPoints: MutableList<PositionPoint>? = null
 
     /**
-     * Sequence of position points describing this location, expressed in coordinate system [Location.CoordinateSystem].
+     * Sequence of [PositionPoint]s describing this location.
```

Comment:
I looked in the CIM docs and found `CoordinateSystem` to be [a real thing](https://zepben.github.io/evolve/docs/cim/cim100/TC57CIM/IEC61968/Common/CoordinateSystem/) that just isn't implemented in the SDK.